### PR TITLE
Add maximum zooming coefficient

### PR DESCRIPTION
**Pull request by BeatC**

### CODE REVIEW

File: modules/UI/videolayout/VideoContainer.js [hunk 1]
```diff
@@ -247,19 +247,30 @@ export class VideoContainer extends LargeContainer {
...
                 containerHeight);
-        } else {
-            return getCameraVideoSize(  width,
-                height,
-                containerWidth,
-                containerHeight);
         }
+
+        const zoomingIndex = containerWidth / width;
+        const maximumZoomingIndex = 1.3;
```
Comment (yanas) [inline modules/UI/videolayout/VideoContainer.js]: Make this configurable from interface_config.

File: modules/UI/videolayout/VideoContainer.js [hunk 2]
```diff
@@ -247,19 +247,30 @@ export class VideoContainer extends LargeContainer {
...
+        let availableWidth = containerWidth;
+        let availableHeight = containerHeight;
+
+        if (zoomingIndex > maximumZoomingIndex) {
+            availableWidth = width * maximumZoomingIndex;
+            availableHeight = height * maximumZoomingIndex;
+        }
+
+        return getCameraVideoSize(  width,
```
Comment (yanas) [inline modules/UI/videolayout/VideoContainer.js]: Can we rename getCameraVideoSize and getDesktopVideoSize to computeCameraVideoSize and computeDesktopVideoSize?

File: modules/UI/videolayout/VideoContainer.js [hunk 3]
```diff
@@ -247,19 +247,30 @@ export class VideoContainer extends LargeContainer {
...
+        const zoomingIndex = containerWidth / width;
+        const maximumZoomingIndex = 1.3;
+        let availableWidth = containerWidth;
+        let availableHeight = containerHeight;
+
+        if (zoomingIndex > maximumZoomingIndex) {
+            availableWidth = width * maximumZoomingIndex;
+            availableHeight = height * maximumZoomingIndex;
+        }
+
```
Comment (yanas) [inline modules/UI/videolayout/VideoContainer.js]: Is there any reason why we don't move this part inside the getCameraVideoSize method? It looks like it's part of it.

**Comment by damencho** [comment]
Obsolete

**Pull request closed without merging**